update readme and remove broken links (#690)
Rebirth-of-the-Night/Rebirth-Of-The-Night: README.md

# Rebirth of the Night

A modpack aimed to bring the survival back in Minecraft and improve many other aspects.

Fear the dark. Defy the odds. Embrace the night. In an ever-changing world, bravery is vital... and arrogance is lethal.

## Community

> Join the community on the [Official RotN Discord](https://discord.gg/rotn). Discuss base ideas, pitch ideas and lore, and keep dabs on recent events and developments.

Reviewer: ```suggestion
Join the community on the [Official RotN Discord](https://discord.gg/rotn). Discuss base ideas, pitch suggestions and lore, and keep tabs on recent events and developments.
```

If you wish to support Rebirth of the Night, consider donating to the [Rebirth of the Night Patreon](https://www.patreon.com/rebirthofthenight).

## Installation

> Client and server files are available through [Curse Forge](https://www.curseforge.com/). See [RotN Modpack on CurseForge](https://www.curseforge.com/minecraft/modpacks/rebirth-of-the-night) for all options.

Reviewer: ```suggestion
Client and server files are available through [CurseForge](https://www.curseforge.com/). See the [RotN Modpack on CurseForge](https://www.curseforge.com/minecraft/modpacks/rebirth-of-the-night) for all options.
```

### Version 2

> Rebirth of the Night is best with the atmosphere provided by **Optifine Shaders** and **Soundtrack.** These mods needs to be installed separately. Follow the [Installation guide here](https://wiki.rebirthofthenight.com/wiki/Installation).

Reviewer: ```suggestion
Rebirth of the Night requires **OptiFine** to run! The **Soundtrack** and **Shaders** are suggested improvements to enhance the atmosphere of the game. All of these mods need to be installed separately. Follow the [Installation guide here](https://wiki.rebirthofthenight.com/wiki/Installation).
```

Reviewer: This is technically what the requirements are, but I'm not sure if you were going for something else @SekretOne

[v2.77.5 Client](https://www.curseforge.com/minecraft/modpacks/rebirth-of-the-night/files/2974800)

[v2.77.5 Server](https://www.curseforge.com/minecraft/modpacks/rebirth-of-the-night/files/2974807)

### Version 3 (Latest Beta)

> As of Version 3, Optifine and SoundTrack are automatically included and **Do not** need to be installed separately.

Reviewer: ```suggestion
As of Version 3, OptiFine and the Soundtrack are automatically included and **do not** need to be installed separately.
```

> [v3.0.Beta-4 Client](https://www.curseforge.com/minecraft/modpacks/rebirth-of-the-night/files/3219146)

Reviewer: ```suggestion
[v3.0 BETA 4 Client](https://www.curseforge.com/minecraft/modpacks/rebirth-of-the-night/files/3219146)
```

> [v3.0.Beta-4 Server Pack](https://www.curseforge.com/minecraft/modpacks/rebirth-of-the-night/files/3219159)

Reviewer: ```suggestion
[v3.0 BETA 4 Server Pack](https://www.curseforge.com/minecraft/modpacks/rebirth-of-the-night/files/3219159)
```
